src/PerfView/CommandLineArgs.cs

```diff
@@ -790,4 +792,16 @@ public static CommandLineLbrFilters Parse(string text)
 
         public override string ToString() => Text;
     }
+
+    /// <summary>
+    /// Flags enum representing the types of Azure credentials that can be used for symbol authentication.
+    /// </summary>
+    [Flags]
+    public enum SymbolsAuthenticationType
```

Comment:
This seems duplicative

Reply:
oh, this is actually another project

Reply:
It is another project, but you're right.  I merged these into one enum definition.